.github/azure-workflows/bootstrap.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+
+# Disable Spotlight indexing. We don't need it, and it might cost performance
+sudo mdutil -a -i off
+
+# Guard against intermittent Travis CI DNS outages
+for host in distfiles.macports.org dl.bintray.com github.com packages.macports.org packages-private.macports.org rsync-origin.macports.org github-production-release-asset-2e65be.s3.amazonaws.com; do
+    dig +short "$host" | sed -n '$s/$/ '"$host/p" | sudo tee -a /etc/hosts >/dev/null
+done
```

Comment:
Have we ever observed DNS issues on GitHub Actions or Azure?

Reply:
Not that I'm aware, but it's not clear whether that's because we have this code. Should we drop it and see if it becomes an issue?

Reply:
Yes, let's try that.